server/routes/label.js

```diff
@@ -0,0 +1,39 @@
+export default (app) => {
```

Comment:
only authentificated user can have access to these routes. we should add some hook